alembic/versions/312fc0f46e81_initial_migration.py

```diff
@@ -0,0 +1,51 @@
+"""Initial migration
+
+Revision ID: 312fc0f46e81
+Revises: 
+Create Date: 2024-10-30 14:13:24.601376
+
+"""
+from typing import Sequence, Union
+
+from alembic import op
+import sqlalchemy as sa
+
+
+# revision identifiers, used by Alembic.
+revision: str = '312fc0f46e81'
+down_revision: Union[str, None] = None
+branch_labels: Union[str, Sequence[str], None] = None
+depends_on: Union[str, Sequence[str], None] = None
+
+
+def upgrade() -> None:
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.create_table('author',
+    sa.Column('id', sa.Integer(), nullable=False),
+    sa.Column('name', sa.String(length=63), nullable=False),
+    sa.Column('bio', sa.String(length=255), nullable=True),
+    sa.PrimaryKeyConstraint('id'),
+    sa.UniqueConstraint('name')
+    )
+    op.create_index(op.f('ix_author_id'), 'author', ['id'], unique=False)
+    op.create_table('books',
+    sa.Column('id', sa.Integer(), nullable=False),
+    sa.Column('title', sa.String(length=255), nullable=False),
+    sa.Column('summary', sa.String(length=1023), nullable=True),
+    sa.Column('publication_date', sa.Date(), nullable=False),
+    sa.Column('author_id', sa.Integer(), nullable=False),
+    sa.ForeignKeyConstraint(['author_id'], ['author.id'], ),
+    sa.PrimaryKeyConstraint('id'),
+    sa.UniqueConstraint('title')
```

Comment:
The `books` table has a unique constraint on the `title` column. Ensure that this constraint aligns with your application's requirements, as it will prevent two books from having the same title.